addons/mail/static/src/core/common/core.scss

```diff
@@ -6,6 +10,10 @@
     min-width: 0;
 }
 
+.o-text-white {
+    color: #FFF;
+}
```

Comment:
Yes, because `text-white` or `bg-black` are theme-specific colors. So `text-white` means color `white` for the current theme, which is white in white mode but black in black mode.

`o-text-whte` is white regardless of theme.